python/gps/gui/util.py

```diff
@@ -60,18 +66,25 @@ def save_to_npz(filename, key, value):
     np.savez(filename, **tmp)
 
 
-def load_pose_from_npz(filename, actuator_name, target_number, data_time):
+def load_pose_from_npz(filename, actuator_name, target_number, data_time,
+        default_ja=np.zeros(7), default_ee_pos=np.zeros(3),
```

Comment:
Can you set these as a constant at the top of the file instead of arguments to the function?